<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>180</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,1,0">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0,0,1">
     <item>
      <widget class="QWidget" name="widget_5" native="true"/>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_6">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Projection Matrix for View:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_5">
       <property name="text">
        <string>PushButton</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="widget_6" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="3">
       <widget class="QLineEdit" name="lineEdit_10"/>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_13"/>
      </item>
      <item row="2" column="0">
       <widget class="QLineEdit" name="lineEdit_11"/>
      </item>
      <item row="0" column="3">
       <widget class="QLineEdit" name="lineEdit_6"/>
      </item>
      <item row="1" column="0">
       <widget class="QLineEdit" name="lineEdit_7"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_12"/>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEdit_8"/>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEdit_4"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit_9"/>
      </item>
      <item row="0" column="0">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="0" column="2">
       <widget class="QLineEdit" name="lineEdit_5"/>
      </item>
      <item row="2" column="3">
       <widget class="QLineEdit" name="lineEdit_14"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_4" native="true"/>
   </item>
   <item>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_4">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QWidget" name="widget_2" native="true"/>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_3">
       <property name="text">
        <string>Ok</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QWidget" name="widget_3" native="true"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>pushButton_6</tabstop>
  <tabstop>pushButton_5</tabstop>
  <tabstop>lineEdit_3</tabstop>
  <tabstop>lineEdit_4</tabstop>
  <tabstop>lineEdit_5</tabstop>
  <tabstop>lineEdit_6</tabstop>
  <tabstop>lineEdit_7</tabstop>
  <tabstop>lineEdit_8</tabstop>
  <tabstop>lineEdit_9</tabstop>
  <tabstop>lineEdit_10</tabstop>
  <tabstop>lineEdit_11</tabstop>
  <tabstop>lineEdit_12</tabstop>
  <tabstop>lineEdit_13</tabstop>
  <tabstop>lineEdit_14</tabstop>
  <tabstop>pushButton_3</tabstop>
  <tabstop>pushButton_4</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
